focus_validator/rules/base_rule_definitions/FV-D003-0003.yaml

```diff
@@ -0,0 +1,4 @@
+check_id: FV-D003-0003
+dimension: InvoiceIssuer
+check:
+  check_unique
```

Comment:
I don't think this is correct. We can't at a line item level validate its unique